client/js/models/archive.js

```diff
@@ -3,7 +3,12 @@ module.exports = {
   namespace: 'archive',
   state: {
     key: null,
-    file: null
+    file: null,
+    numPeers: 0,
+    signalhubs: [
+      'signalhub.mafintosh.com',
+      'signalhub.dat.land'
```

Comment:
Ok, after discussing it with yoshua, he has convinced me that instead of a rehydration object printed into the html document, we should make an xhr call to rehydrate the app. That way the html document pieces become extremely cache-able.